yui/src/article/js/form.js

```diff
@@ -299,22 +299,13 @@ Y.extend(FORM, Y.Base,
         /**
          * Removes all dynamically opened forms.
          *
+         * Removed code from this method because it caused issues:
+         * when the editor menu option is clicked (like paragraph style)
+         * the form gets removed.
+         *
          * @method removeAllForms
          */
         removeAllForms: function() {
-            Y.log('Removing all forms', 'info', 'Form');
-
-            Y.all(SELECTORS.POSTS + ' ' + SELECTORS.FORM_REPLY_WRAPPER).each(function(node) {
-                // Don't removing forms for editing, for safety.
-                if (!node.ancestor(SELECTORS.DISCUSSION_EDIT) && !node.ancestor(SELECTORS.POST_EDIT)) {
-                    node.remove(true);
-                }
-            });
-
-            var node = Y.one(SELECTORS.ADD_DISCUSSION_TARGET);
-            if (node !== null) {
-                node.empty();
-            }
         },
```

Comment:
Hi, what is the reason for having an empty function? "removeAllForms: function(){}"

Reply:
Code in other places calling this method. I don't think it's a major issue to leave this function skeleton. If you or Brendon disagree I'll remove it completed.

Reply:
Have you tested this? So if you click reply on different posts, does everything still behave as normal? If other functions call it, think it might be better to leave it as a skeleton function provided that everything still works as expected

Reply:
Tested now:
New topic works. Reply to topic works. BUT reply to a reply does not show advanced editor. Need to fix that issue.

Reply:
Please flag with BSA that this is an issue